test/gen_rpc.coverspec

```diff
@@ -1,5 +0,0 @@
-{nodes, ['gen_rpc_master@127.0.0.1', 'gen_rpc_slave1@127.0.0.1']}.
```

Comment:
`rebar3` doesn't support cover spec.s (at least that's what the UI says). At the same time we're not exposing this info anywhere and there are more modern approaches to this problem.

Reply:
👍